packages/instantsearch.js/src/components/Pagination/Pagination.tsx

```diff
@@ -113,7 +113,7 @@ function Pagination(props: PaginationProps) {
 
         {props.showLast && (
           <PaginationLink
-            ariaLabel="Last"
+            ariaLabel={`Last Page of ${props.nbPages}`}
```

Comment:
this sentence reads a little weird, has that been tested?

Reply:
I see a variant in the [US Web Design System](https://designsystem.digital.gov/components/pagination/#accessibility), which I think would work better:

> Voice “last page” on the last page’s link in a bounded set. Use aria-label="last page, page [#]" to voice that the last element in the navigation is the last page in the paginated section.